comments/classified.csv

```diff
@@ -1878,7 +1863,6 @@ In addition #code is only available in the scope of this function. I would need
 This method wraps IUnknown in a managed object. This has the effect of incrementing the reference count of the COM component. The reference count will be decremented when the runtime performs garbage collection on the managed object that represents the COM object,0,0.5
 "In SafeHandles.cs, there are three calls to Marshal.GetObjectForIUnknown() that create a local variables that disappear into the ether and never have Marshal.ReleaseComObject() called on them (until garbage collection).  In my tests, adding a Marshal.ReleaseComObject() call to each of these will fix the issue.  For example",0,0.5
 "Like I said, I believe this fixes the issue (presumably, the right way) and should allow this PR to be merged.  Please let me know your thoughts.  It'd be great if we could get this merged soon.  Thanks",0,0.5
-@github,0,0.5
```

Comment:
Same with this one, this would be @-mentioning someone.

Reply:
Fixed!